experimenter/experimenter/experiments/models.py

```diff
@@ -798,6 +798,29 @@ def allocate_bucket_range(self):
             ),
         )
 
+    @property
+    def feature_has_live_multifeature_experiments(self):
+        """Live multifeature experiments that share a feature with this experiment.
+
+        Returns:
+            A list of experiment slugs.
+        """
+        matching = set()
+        live_experiments = NimbusExperiment.objects.filter(
+            status=self.Status.LIVE,
+            application=self.application,
+        )
+        if len(live_experiments.all()) >= 0:
+            for feature in self.feature_configs.all().values_list("slug", flat=True):
+                experiments = (
+                    live_experiments.annotate(n_feature_configs=Count("feature_configs"))
+                    .filter(n_feature_configs__gt=1)
+                    .filter(feature_configs__slug=feature)
+                    .all()
+                )
+                matching.update([e.slug for e in experiments])
```

Comment:
And then you can just do the same thing you did above to pull the slugs out and then you don't need the set either 

```py
.values_list("slug", flat=True).distinct()
```

Reply:
Oh actually and prolly wanna throw a `.order_by('slug')` in here too so the order doesn't jump around in the UI, since it's doing a join and the order is just 🤷‍♀️ 🤷‍♀️ 🤷‍♀️